### PR TITLE
Release 0.2.2

### PR DESCRIPTION
**Pull request by adietish**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,12 +1,31 @@
+buildscript {
+    ext.ideaVersion = ideaVersion
+    /*
+     * for more complex conversion ideaVersion -> sinceIdeaBuild
+     * see https://github.com/rhdunn/xquery-intellij-plugin/blob/master/build.gradle#L1-L47
+     */
+    def since = ideaVersion =~ /IC-20([0-9][0-9])\.([1-3])(\.[0-9]+)?$/
```
Comment (jeffmaury) [inline build.gradle:7]: Why don't you use ideaVersion.substring(3, 6)

File: gradle.properties [hunk 2]
```diff
@@ -1,9 +1,17 @@
-ideaVersion=IC-2018.3
+ideaVersion=IC-2019.3
```
Comment (jeffmaury) [inline gradle.properties:1]: Seems to me that your lowest version is now 2019.3 so IJ.yml does not match

File: build.gradle (hunk 1 above)
Comment (adietish) [reply build.gradle:7]: need to also remove the '.' (IC-2019.3 -> 193)
Btw. this PR is about releasing a fix-version. I'll rebase it once #243 is merged.

File: gradle.properties (hunk 2 above)
Comment (adietish) [reply gradle.properties:1]: IJ.yml was corrected, see here: https://github.com/redhat-developer/intellij-kubernetes/pull/245/files#diff-d2a86296b50b44799992fba964e1e33c7d03d00535fd5e24388d3981f6415332L17

Comment (jeffmaury) [reply gradle.properties:1]: It's not aligned because IJ.yml used 2019.1 as the lowest

Comment (adietish) [reply gradle.properties:1]: sorry, my bad. Will be corrected in #243 and this one rebased.

**Comment by adietish** [comment]
@jeffmaury please approve/reject.

**Review by jeffmaury — APPROVED** [review]
LGTM

**Comment by adietish** [comment]
published to marketplace: ttps://studio-jenkins-csb-codeready.apps.ocp4.prod.psi.redhat.com/job/Adapters/job/IntelliJ/job/intellij-kubernetes-build/50

**Pull request merged**